web-registry/src/lib/ecocredit.ts

```diff
@@ -130,3 +131,11 @@ const getAccountEcocreditsForBatch = (
     `${ledgerRestUri}/regen/ecocredit/v1alpha1/batches/${batchDenom}/balance/${account}`,
   );
 };
+
+export const getClassInfo = async (
+  class_id: string,
+): Promise<AxiosResponse<QueryClassInfoResponse>> => {
+  return axios.get(
+    `${ledgerRestUri}/regen/ecocredit/v1alpha1/classes/${class_id}`,
+  );
+};
```

Comment:
Minor nit: you don't need `async` for this function as-is

I thought through this a bit and figured for our use, it makes sense to `await` the response data and return within this function (so you don't have to dig through `response.data` every time you use) +  a basic trycatch for error handling. Something like 

```suggestion
export const getClassInfo = async (
  class_id: string,
): Promise<QueryClassInfoResponse> => {
  try {
    const { data } = await axios.get(
      `${ledgerRestUri}/regen/ecocredit/v1alpha1/classes/${class_id}`,
    );
    return data;
  } catch (err) {
    throw new Error(`Error fetching class info: ${err}`);
  }
};
```